### ISSUE TITLE
code understanding

### ISSUE THREAD

**Issue by as485** [hamzafaisaljarral/scoop#1]
Hi there, When I was learning and trying to understand the function "find_referenced_templates(ast)" in scoop/shoptutorial/lib/python3.6/site-packages/cms/admin/placeholderadmin.py, I was confused about the description of the its delivered function. My understanding of this code snippet is "iterates through the glob nodes". Do I understand that right?

**Comment by hamzafaisaljarral** [comment]
Yup you have got that right but for better understanding you have to look into django-shop documentation and look into django-cms documentation as well .

**Comment by as485** [comment]
Thanks for your response very much. I will read the code more carefully.
